src/grafana_dashboards/postgresql-metrics.json

```diff
@@ -2191,10 +2200,10 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "pg_locks_count{datname=~\"$datname\", instance=~\"$instance\", mode=~\"$mode\"} != 0",
+          "expr": "irate(pg_stat_database_tup_updated{datname=~\"$datname\", instance=~\"$instance\"}[5m])",
```

Comment:
this kind of weird diffs are mostly repositioning of panels, they're not really being replaced.